contracts/Comptroller.sol

```diff
@@ -475,7 +478,7 @@ contract Comptroller is
         uint256 borrowBalance = VToken(vTokenBorrowed).borrowBalanceStored(borrower);
 
         /* Allow accounts to be liquidated if the market is deprecated or it is a forced liquidation */
-        if (skipLiquidityCheck || isDeprecated(VToken(vTokenBorrowed))) {
+        if (skipLiquidityCheck || isDeprecated(VToken(vTokenBorrowed)) || isForcedLiquidationEnabled[vTokenBorrowed]) {
```

Comment:
Let's remove the `isDeprecated` function. We are replacing it with the flag `isForcedLiquidationEnabled`, so we can enable/disable forced liquidations in a arbitrary way with a VIP

Reply:
Removed, cbd9b18a99c4e1f92bf9404e88fceb8ebc36d55f